### ISSUE TITLE
Carousel into carousel and scrolling

### ISSUE THREAD

**Issue by SotirisValogiannis** [smashingboxes/OwlCarousel2#890]
I have an issue that I can't figure out what could be the problem. I have a carousel where on click I add a new carousel with ajax in a newly created .owl-item, you can check it here: http://goo.gl/Th4tCa

The problem is that in the inner carousel clicking the arrows in the top (beside close text) the content doesn't move to the next slide, while I use the same code on other page and works http://goo.gl/6uFcRb
I am using the following code to jump to the next item:

    $('.slides').on('click', '.tile-full-wrapper .photos-index .next', function(e){
		e.preventDefault();
		$('.tile-full').data('owlCarousel').next();
		changeCurrentThumbIndex();
	});

What I have noticed is that if I have more items in the main carousel the inner carousel works or partially works depending of the number of main carousel items. The dummy items I have added in main carousel have 1px width, so it's not that the stage getting bigger.

You can add dummy items in console using `addDummyTiles( number )` to see what I mean.

Any ideas what could be the issue?

Many thanks

**Comment by SotirisValogiannis** [comment]
From some experiments I made I found out that when I click the arrows the navigation that fired to navigate through the slides is not the one the inner carousel but  the main one. Why that happens?

In ` Navigation.prototype.to` the `this` object is the main navigation and the `this._core` the Owl object of main carousel. Any ideas on what I separate these or any idea for any issue in my structure?

Thanks!
